### PR TITLE
Feature/loading message

### PR DESCRIPTION
**Pull request by Jay-zar**
This adds a basic loading message while the API call is fetching data. Loading state is handled as a context on the Adopt page with state set in ZipCodeInput.

### CODE REVIEW

**Review by tarikutk — CHANGES_REQUESTED** [review]
is it for a reason that you deleted all the lines in .env.example ? or may be it was just because the branch was not created from the lates main branch. also in adopt page you replaced < Search/> with < ZipCodeInput/>. so you may want to revert that because the < Search/> has the <Zipcodeinput/> and also buttons that will helps us filter

File: src/components/ZipCodeInput.js [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 import { PetsContext } from "../context"
+import { LoadingContext } from "../context"
 import { SearchContext } from "../context"
 
 const ZipCodeInput = () => {
   //using PetsContext and SearchContext
   const { setPetList } = useContext(PetsContext)
+  const { setIsLoading } = useContext(LoadingContext)
+  const [zipcodeFormValues, setZipcodeFormValues] = useState("")
+
```
Comment (MarissaLZ) [inline src/components/ZipCodeInput.js:15]: These lines can be deleted. I removed them when I created the <SearchContext/> which takes care of the zipcode value now.

File: src/pages/Adopt.js [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
+import LoadingMessage from "../components/LoadingMessage"
 import Search from "../components/Search"
 
+
 const Adopt = () => {
+  const [isLoading, setIsLoading] = React.useState(false);
   return (
-    <div>
-      <Search />
-      <PetList />
```
Comment (MarissaLZ) [inline src/pages/Adopt.js:9]: These two lines are needed to have the <Search/> component that allows the user to select a zip code and pet type. <ZipCodeInput/> was moved to Search.js in a previous pull.

File: src/pages/Adopt.js [hunk 3]
```diff
@@ -1,13 +1,22 @@
 import React from "react"
+import { useState } from "react"
+import ZipCodeInput from "../components/ZipCodeInput"
```
Comment (MarissaLZ) [inline src/pages/Adopt.js:3]: This line can be removed since `<ZipCodeInput/>` was moved to Search.js

**Review by MarcusCHill — APPROVED** [review]
Loading works well, no errors.  To address warning message Adopt.js imports useState so the React. can be ommited on line 10 or the import can be omitted.

**Review by tarikutk — APPROVED** [review]

**Review by MarissaLZ — APPROVED** [review]

**Pull request merged**